buildmaster/H1_1JET_319GEV_290PB-1_DIF/metadata.yaml

```diff
@@ -0,0 +1,120 @@
+setname: 'H1_1JET_319GEV_290PB-1_DIF'
+
+nnpdf_metadata:
+  nnpdf31_process: "1JET"
+  experiment: "H1"
+
+arXiv:
+  url: "https://arxiv.org/abs/1611.03421"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1496981"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1496981"
+  version: 1
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "double differential cross section w.r.t. q^2 and pT", label: '$\frac{d^2\sigma}{dq^{2}dpT}$', units: "$pb/GeV^3$"}
+    observable_name: PTQ2
+    process_type: JET
+    ndata: 48
+    tables: [1, 2, 3, 4, 5, 6, 7, 8]
+    kinematics:
+      variables:
+        pT: {description: "mean transverse momentum", label: '$pT$', units: "$GeV$"}
+        q_sqr: {description: "momentum transferred squared", label: '$q^2$', units: "$GeV^2$"}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics.yaml
+    data_central: data.yaml
+    data_uncertainties:
+      - uncertainties.yaml
+    kinematic_coverage: [pT, q_sqr, sqrt_s]
```

Comment:
Instead of q_sqr, use Q2 since this is more or less the standard notation

Reply:
done